<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reOrderDock</class>
 <widget class="QWidget" name="reOrderDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reOrderDock</string>
  </property>
  <widget class="QComboBox" name="reseauComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="exutoireComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point network layer :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exutoire layer :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output layer :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launchButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
